modules/sfp_dnsbrute.py

```diff
@@ -4,17 +4,17 @@
 # Purpose:      SpiderFoot plug-in for attempting to resolve through brute-forcing
 #               common hostnames.
 #
-# Author:      Steve Micallef <steve@binarypool.com>
+# Author:      TheTechromancer
```

Comment:
Understand this is a substantial re-write, but would prefer you add yourself and comment that it was a substantial re-write.